src/lib/web-publisher-status.js

```diff
@@ -15,10 +16,21 @@ class PublisherStatus {
       GROUP BY publisher.id
     `;
 
-    const res = await client.query(publisherSqlQuery);
+    let res;
 
-    client.release();
-    return res.rows;
+    try {
+      res = await client.query(publisherSqlQuery);
+    } catch (err) {
+      console.warn(err);
+    } finally {
+      client.release();
+    }
+
+    if (res){
+      return res.rows;
```

Comment:
I'm not sure if calling .rows after a .release() causes issues on this DB driver but presumabley you've seen it be ok. We should bear it in mind if we start to get odd errors when under load tho.

Reply:
I'd hope it wouldn't be garbage collected as long as the reference exists.